images/Document about me.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
```

Comment:
It looks like this file isn't used at all on your webpage. It would be good to remove files that are not used.